andela/vidar-ah-frontend
# #164139638 Setup react router version 4 (react-router-dom)

#### What does this PR do?
Sets up routing functionality for the application.

#### Description of Task to be completed?
- install and use react-router-dom for routing.
- create the folder structure for the components and pages.
- create the home, log in and 404 components.
- enable routing to the homepage, login page and 404 pages.

#### How should this be manually tested?
- run `npm run start:dev`, this will launch the app and would load up the landing page in your default browser.
- navigate to this route `http://localhost:3000/login` to see the login page.
- navigate to a route that does not exist and you would get a 404 page.

#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
#164139638
